packages/mux-uploader-react/src/useObjectPropEffect.ts

```diff
@@ -55,7 +55,7 @@ export const useObjectPropEffect = <T extends { [k: string]: any }, V>(
   updateValue = defaultUpdateValue,
   hasChanged = defaultHasChanged
 ) => {
-  return useEffect(() => {
+  return React.useEffect(() => {
```

Comment:
nitpick(non-blocking): instead of changing usage of hooks, consider a module-level destructuring, e.g.:
```ts
import React from 'react';
const { useEffect } = React;
```